create-custom-map-style/amplify/backend/auth/createcustommapstyled09deb85/parameters.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Files under `amplify` are created by the CLI, right? If so, do they need to be checked in?

Reply:
I think it should probably be omitted, but check the Amplify docs to confirm.